### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by gdmyd** [Rain9876/Auto-Code-Generator#1]
Hello! After reading your paper, i want to do all experiments you had done. Could i do it with this repository? Thank you!

**Comment by Rain9876** [comment]
Yes, all experiments can be done with the python source code from the Microsoft CodeXGLUE. The pretrain data comes from Google BigQuery github data if you need.
Check the Resource.md in the data processing folder for the data information.
